samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java

```diff
@@ -44,6 +44,10 @@ public static void main(String[] args) throws IOException, InterruptedException
     String branchName =
         String.format(
             "projects/%s/locations/global/catalogs/default_catalog/branches/0", projectId);
+    // TO CHECK ERROR HANDLING USE INVALID CATALOG IN REQUEST
```

Comment:
nit: can this be made into lower case? e.g. `To check error handling, use an invalid catalog in request` or something similiar

Reply:
Refactored.